msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/SignUpStates.kt

```diff
@@ -586,8 +667,16 @@ class SignUpAttributesRequiredState internal constructor(
 class SignInAfterSignUpState internal constructor(
     override val continuationToken: String?,
     override val username: String,
-    private val config: NativeAuthPublicClientApplicationConfiguration
+    val config: NativeAuthPublicClientApplicationConfiguration
 ) : SignInAfterSignUpBaseState(continuationToken, username, config) {
+
+    constructor(parcel: Parcel) : this(
+        parcel.readString(),
```

Comment:
This and line 675 should be swapped, right? The first parcel.read entry should handle the first constructor parameter, which is nullable. The second handles a non-null parameter.

Reply:
Oh right, I see 👍